runtime/doc/news.txt

```diff
@@ -87,6 +87,13 @@ TREESITTER
   capture IDs to a list of nodes that need to be iterated over. For
   backwards compatibility, an option `all=false` (only return the last
   matching node) is provided that will be removed in a future release.
+• |vim.treesitter.language.get_filetypes()| always includes the {language}
+  argument in addition to explicitly registered filetypes.
```

Comment:
returning extra data is not usually considered breaking.

Reply:
It's not extra, it's returning where it used to not return -- and it's reasonable to assume that some people relied on the `nil` to be able to, e.g., do something else instead of starting treesitter. We could claim this falls under "treesitter is experimental", but I thought it better to err on the side of caution and give a shoutout here.